libindy/src/api/wallet.rs

```diff
@@ -519,6 +524,50 @@ pub extern fn indy_delete_wallet(command_handle: i32,
     res
 }
 
+/// Generate wallet master key.
```

Comment:
Update description:

Generate wallet master key. Returned key is compatible with "RAW" key derivation method. It allows to avoid expensive key derivation for use cases when wallet keys can be stored in a secure enclave.